custom/icds_reports/utils/aggregation_helpers/awc_location.py

```diff
@@ -100,6 +100,22 @@ def aww_query(self):
             ucr_aww_tablename=self.ucr_aww_tablename
         )
 
+    def ls_query(self):
```

Comment:
Why not just change this line here: https://github.com/dimagi/commcare-hq/blob/master/custom/icds_reports/utils/aggregation_helpers/awc_location.py#L87

Reply:
Hey,Because for updating the ls_name where condition is to be changed at line 97.